src/layers/geojson-layer/geojson-layer.js

```diff
@@ -379,19 +380,21 @@ export default class GeoJsonLayer extends Layer {
     const {fixedRadius} = this.meta;
     const radiusScale = this.getRadiusScaleByZoom(mapState, fixedRadius);
     const zoomFactor = this.getZoomFactor(mapState);
+    const eleZoomFactor = this.getElevationZoomFactor(mapState);
+
     const {visConfig} = this.config;
 
     const layerProps = {
       lineWidthScale: visConfig.thickness * zoomFactor * 8,
-      elevationScale: visConfig.elevationScale,
+      elevationScale: visConfig.elevationScale * eleZoomFactor,
       pointRadiusScale: radiusScale,
       lineMiterLimit: 4
     };
 
     const updateTriggers = {
       getElevation: {
         heightField: this.config.heightField,
-        heightScale: this.config.heightScale,
+        heightScaleType: this.config.heightScale,
```

Comment:
new naming convention?

Reply:
this is just to avoid conflict with deck.gl layer props, for updateTriggers it doesn't really matter of the name, becuase it's a shallow comparison between `newUpdateTriggers. heightScaleType` and `oldUpdateTriggers. heightScaleType`